App.tsx

```diff
@@ -31,7 +31,7 @@ export default class App extends React.Component {
     }
 
     return (
-      <View style={styles.container}>
+      <View>
```

Comment:
Why did you remove the styling here?

Reply:
Having a container on the whole page wasn't letting me put the recipe picture as a banner that covered the whole screen